### ISSUE TITLE
Naming of QnAMaker vs LUIS projects

### ISSUE THREAD

**Issue by nrobert** [Microsoft/botbuilder-dotnet#324]
**Report:**
Currently we have on the library side two projects for LUIS and QnAMaker:

- `Microsoft.Bot.Builder.Ai` (containing QnAMaker but also a TranslationMiddleware)
- `Microsoft.Bot.Builder.Luis` for LUIS

Why is it split like that?

I find this naming confusing, especially as people often think that LUIS and/or QnAMaker are part of MS Bot Framework (when you look at some StackOverflow questions).

**Proposal for the library projects:**
Shouldn't we group all Cognitive services integration under a global namespace, like they are currently doing on NuGet packages created to consume those services with `Microsoft.Azure.CognitiveServices...` which will replace `ProjectOxford` packages?

It could be `Microsoft.Bot.Builder.Ai` (or `Microsoft.Bot.Builder.CognitiveServices`) for the namespace root, and we would have a project for each "Cognitive category" (those from [Azure website](https://azure.microsoft.com/en-us/services/cognitive-services)):

- Microsoft.Bot.Builder.Ai.Knowledge (containing QnAMaker)
- Microsoft.Bot.Builder.Ai.Language (for LUIS and Translation)
...

What do you think about that?
I'm not sure that the category would be necessary because some Cognitive services may not be useful in a bot case, but currently the split between QnAMaker + Translation vs LUIS is strange (in my opinion)

**Comment by garypretty** [comment]
This is something that @cleemullins agree discussed. I agree that the split right now isn't good and doesn't make sense. I've got it on my list in the next day or so to split out QnA and translation info their own projects given that I think @cleemullins prefers to keep luis on its own so we need to be consistent. Also make package management / testing easier if we keep them isolated rather than all in one bundle. Look out for the PR soon.

**Comment by nrobert** [comment]
Great.

For the naming, do you think we could keep the `Ai` part to group all cognitive services?

- Microsoft.Bot.Builder.Ai.QnAMaker
- Microsoft.Bot.Builder.Ai.LUIS
- Microsoft.Bot.Builder.Ai.Translate
...

**Comment by garypretty** [comment]
Yes. That's the plan. Which makes sense as not everything will necessary be cognitive services

**Comment by garypretty** [comment]
@cleemullins Are you happy for me to proceed with splitting out as mentioned above?  The consensus so far does seem to suggest that the current setup is confusing. If so, let me know and I will progress and submit a PR.

**Comment by garypretty** [comment]
@cleemullins now that the PR has been merged we should be able to close this issue
